pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -28,89 +20,93 @@ fi
 
 # Install minimum dependencies
 if command -v apt-get >/dev/null 2>&1; then
-	DEBIAN_FRONTEND=noninteractive
-	export DEBIAN_FRONTEND
-	apt-get update
-	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ]; then
-		if ! command -v sshfs >/dev/null 2>&1; then
-			apt-get install -y sshfs
-		fi
+	pkgs=
```

Comment:
This is just personal preference, but a dangling `=` sign looks to me like the right side might have been forgotten, or accidentally deleted. So I prefer to explicitly set to the empty string:

```suggestion
	pkgs=""
```

Reply:
Fixed